docs/samples/Microsoft.ML.Samples/Dynamic/DataOperations/SaveAndLoadFromText.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.ML;
+
+namespace Samples.Dynamic
+{
+    public static class SaveAndLoadFromText
+    {
+        public static void Example()
+        {
+            // Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
+            // as a catalog of available operations and as the source of randomness.
+            // Setting the seed to a fixed number in this example to make outputs deterministic.
+            var mlContext = new MLContext(seed: 0);
+
+            // Create a list of training data points.
+            IEnumerable<DataPoint> dataPoints = GenerateRandomDataPoints(10);
```

Comment:
>GenerateRandomDataPoints [](start = 48, length = 24)

no need to use GenerateRandomDataPoints function when training is not needed. you can just define 5 data points inline and keep the sample file shorter.